### PR TITLE
SFB-165: concept scheme editor go back to form button

### PR DESCRIPTION
**Pull request by ZijneDwaasheid**
## ID
SFB-165

 ## Description
Instead of going back to the index page of the formbuilder, we now have the option to go back to form where we clicked the button 'codelijsten'. If we landed on the codelijsten page on a different way, not from a form, the return button will bring you to the index of the formbuilder.

 ## Type of change

 - [ ] Bug fix
 - [x] New feature
 - [ ] Breaking change
 - [ ] Other

 ## How to test

Try to do something that causes something that is not expected, like described in the description of this pr.

### CODE REVIEW

File: app/services/formbuilder-id-service.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  setFormbuilderId(id) {
+    this.formbuilderId = id;
+  }
+
+  getFormbuilderId() {
+    return this.formbuilderId;
+  }
+
+  clearFormbuilderId() {
+    this.formbuilderId = EMPTY_FORMBUILDER_ID;
```
Comment (JonasVanHoof) [inline app/services/formbuilder-id-service.js:17]: Would just set it to null

File: app/services/formbuilder-id-service.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import Service from '@ember/service';
+
+const EMPTY_FORMBUILDER_ID = '';
```
Comment (JonasVanHoof) [inline app/services/formbuilder-id-service.js:3]: Can be removed than as set to null in clear method

File: app/services/formbuilder-id-service.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+export default class FormbuilderIdServiceService extends Service {
```
Comment (JonasVanHoof) [inline app/services/formbuilder-id-service.js:5]: Not a fan of the service name tbh..

* formbuilder => the application
* id => managing the application instance id's? (blowing things up 😁 )
* service is twice in the name

Did you check the sticky router params that @aatauil mentioned?

A service is a little to big for just going back one route in one place IMO if the sticky query router param does not work. Not sure if this is something but we can maybe store it in the storage of the browser? @aatauil

**Review by JonasVanHoof — COMMENTED** [review]
It works but some remarks that should be checked first

File: app/services/formbuilder-id-service.js (hunk 3 above)
Comment (ZijneDwaasheid) [reply app/services/formbuilder-id-service.js:5]: yes will check on that

**Comment by JonasVanHoof** [comment]
@ZijneDwaasheid  this one can be closed no?

**Comment by ZijneDwaasheid** [comment]
> @ZijneDwaasheid this one can be closed no?

yes indeed

**Pull request closed without merging**